src/react-components/preferences-screen.js

```diff
@@ -154,18 +153,20 @@ export class NumberRangeSelector extends Component {
               this.setState({ displayValue: sanitizedInput, digitsFromUser: countDigits(sanitizedInput) });
               const numberOrReset = isNaN(parseFloat(sanitizedInput)) ? undefined : parseFloat(sanitizedInput);
               this.props.setValue(numberOrReset);
+              this.sliderRef.current?.setValue(numberOrReset);
             }}
           />
         </div>
         <Slider
+          ref={this.sliderRef}
           step={this.props.step}
           min={this.props.min}
           max={this.props.max}
-          value={currentValue}
+          defaultValue={currentValue}
```

Comment:
This is again quite suspicious, and misleading, since the value of this prop is only used when the component is first mounted (to set the initial state), so changing it has no effect. It works anyway since the ref is actually what is used to update the internal state (imperatively)